Sources/ProjectDescription/FileCodeGen.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  FileCodeGen.swift
+//  FileCodeGen
+//
+//  Created by Mahadevaiah, Pavan | Pavan | ECMPD on 2021/09/14.
+//
```

Comment:
minor: Worth removing this for consistency with the rest of the codebase where all header comments are omitted